src/components/Calculator.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Let's make sure we don't use disable linter rules. These are here to help us write clean code.